dbt/adapters/starrocks/connections.py

```diff
@@ -15,6 +15,9 @@
 
 from contextlib import contextmanager
 
+import json
+from json.decoder.json import JSONDecodeError
```

Comment:
@Hatlassian After running `python setup.py develop`, executing `dbt debug` generates the following error:
`ModuleNotFoundError: No module named 'json.decoder.json'; 'json.decoder' is not a package`

Reply:
`from json import JSONDecodeError` can work